css/style.css

```diff
@@ -266,12 +269,18 @@ input[type="checkbox"] {
 /* ----------- UI - Graph ----------- */
 
 .vis {
-  display: grid;
```

Comment:
current `.vis` has `1fr 2fr`. We can change this ratio anytime, `1fr 3fr` gives more space to `#visualization`.